drivers/platform/chibios/chibios_alloc.c

```diff
@@ -39,6 +39,7 @@
 
 #include "no_os_alloc.h"
 #include "hal.h"
+#include <string.h>
```

Comment:
I would add this line to a different commit and a Fixes tag would be nice for both commits